dev/breeze/src/airflow_breeze/utils/publish_docs_helpers.py

```diff
@@ -112,7 +112,7 @@ def process_package_filters(
         return available_packages
 
     expanded_short_form_packages = get_provider_name_from_short_hand(packages_short_form)
-    package_filters = list(package_filters + expanded_short_form_packages)
+    package_filters = package_filters + expanded_short_form_packages
```

Comment:
```suggestion
    package_filters = [*package_filters, *expanded_short_form_packages]
```

Reply:
It would be a good idea to rewrite the logic below so we only iterate through this once and can use `itertools.chain` here instead.